src/components/backend-ai-edu-applauncher.ts

```diff
@@ -331,7 +339,8 @@ export default class BackendAiEduApplauncher extends BackendAIPage {
       const templateId = sessionTemplates[0].id; // NOTE: use the first template. will it be okay?
       try {
         const mounts = await globalThis.backendaiclient.eduApp.get_mount_folders();
-        const resources = mounts ? {mounts} : {};
+        const projects = await globalThis.backendaiclient.eduApp.get_user_projects();
+        const resources = mounts ? {mounts, ...this.resources, 'group_name': projects[0]['name']} : {...this.resources, 'group_name': projects[0]['name']};
```

Comment:
this line can occur `TypeError` if the length of the projects is 0.

Reply:
Could you recommend me how to log or raise exception when `project` is empty?
Cause the backend ensures that every user has a project. it should be an error if it is empty.